charts/alfresco-connector-ms365/tests/deployment-connector-ms365_test.yaml

```diff
@@ -0,0 +1,29 @@
+---
+suite: test connector-ms365 deployment
+templates:
+  - deployment-connector-ms365.yaml
+  - config-connector-ms365.yaml
+tests:
+  - it: should have basic metadata in place
+    set: &values
+      postgresql:
+        enabled: true
+    asserts:
+      - equal:
+          path: metadata.name
+          value: RELEASE-NAME-alfresco-connector-ms365
+        template: deployment-connector-ms365.yaml
+
+  - it: should render cpu and memory limits
```

Comment:
Rendering of default resources boundaries is already checked by checkov rules. Instead you might want to check custom values are rendered properly.

Reply:
Why keeoing the yaml alias `&values` if it's empty?